### ISSUE TITLE
dvc run warnings

### ISSUE THREAD

**Issue by dmpetrov** [dataversioncontrol/dvc#286]
There are some messages that we don't need:
```
$ dvc run python code/xml_to_tsv.py data/Posts.xml data/Posts.tsv python
File python from argv is outside of git directory and cannot be traced
File data/Posts.tsv from argv is outside of git directory and cannot be traced
File python from argv is outside of git directory and cannot be traced
```

**Comment by efiop** [comment]
This is direct cause of that 'magic class' issue, I have an upcoming patch for this.

**Comment by efiop** [comment]
Fixed with #356
